code/modules/client/preference/preferences_mysql.dm

```diff
@@ -261,21 +260,19 @@
 		player_alt_titles = params2list(query.item[58])
 		organ_data = params2list(query.item[59])
 		rlimb_data = params2list(query.item[60])
-		nanotrasen_relation = query.item[61]
-		speciesprefs = text2num(query.item[62])
+		speciesprefs = text2num(query.item[61])
```

Comment:
Are you sure? If so, why do you think that? Above this, we eliminate nanotrasen_relation from the list. Additionally, there is SQL in the OP to drop the col.